### PR TITLE
Fix: Rule can't find reference of `create` function

### LINKED ISSUE #106: `getRuleInfo` not working
[`getRuleInfo`](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/02c1f347f2f0881e364724a4fce69e9da2ce9313/lib/utils.js#L87) seems not working if rule write in this way

```js
'use strict';

const create = () => {};

module.exports = {create};
```

See https://github.com/sindresorhus/eslint-plugin-unicorn/blob/e7e25b82526e42970043c39a512cbdfb889b7cb7/rules/better-regex.js#L107

I dig a little bit, [this line](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/02c1f347f2f0881e364724a4fce69e9da2ce9313/lib/utils.js#L32) , `scopeManager.acquire(node)` is null, so it will use `globalScope`. If I change to `scopeManager.globalScope.childScopes[0]` it will work.

### PR DESCRIPTION
**Pull request by fisker**
Turns out if `create` is a reference, the global scope don't have the reference not only in `nodejsScope`, also broken if `sourceType` is set to `'module'`.

To find the correct reference, I'm searching all `childScopes`, I hope it's fine to do this.

Fixes #106

### CODE REVIEW

**Comment by fisker** [comment]
@aladdin-add  No CI running?

File: tests/lib/utils.js [hunk 1]
```diff
@@ -122,6 +122,31 @@ describe('utils', () => {
...
+        { ignoreEval: true, ecmaVersion: 6, sourceType: 'module' },
+      ]) {
+        const ast = espree.parse(`
+          const create = () => {};
+          const meta = {};
+          module.exports = { create, meta };
+        `, { ecmaVersion: 6 });
+        const expected = {
+          create: { type: 'Identifier' },
+          meta: { type: 'Identifier' },
```
Comment (fisker) [inline tests/lib/utils.js:138]: This should still break rules like [this](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/02c1f347f2f0881e364724a4fce69e9da2ce9313/lib/rules/meta-property-ordering.js#L42), It should return the reference. But I'm not going to work on it in this PR.

**Comment by aladdin-add** [comment]
ci was not working somehow ╮(￣▽￣)╭, we are considering switching to gh actions. refs #104

**Review by aladdin-add — APPROVED** [review]
LGTM, thanks!

**Review by not-an-aardvark — APPROVED** [review]

**Pull request merged**